### ISSUE TITLE
dist folder not found under bower directory

### ISSUE THREAD

**Issue by AmbrocioIsaias2808** [philippfrenzel/yii2fullcalendar#158]
The dist folder cannot be found:
![image](https://user-images.githubusercontent.com/44043979/116108527-9706f000-a679-11eb-89c4-9c20c605e324.png)

And appearently when installing the folder is not generated:
![image](https://user-images.githubusercontent.com/44043979/116108795-d9303180-a679-11eb-9a00-8fae4053e390.png)

![image](https://user-images.githubusercontent.com/44043979/116108956-fd8c0e00-a679-11eb-8173-72df9dadc885.png)

**Comment by gledsoneduardo** [comment]
composer require philippfrenzel/yii2fullcalendar "3.9.0"

**Comment by dsdobrzynski** [comment]
While using an older version resolves the error, it doesn't fix the underlying issue in the latest tagged version: 4.0.2. Are there plans to fix it in that version?

**Comment by mkraibt** [comment]
Yes still has the dist folder missing error with the latest version.
